jobs/release.Jenkinsfile

```diff
@@ -71,13 +71,13 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
 
         // Fetch metadata files for the build we are interested in
         stage('Fetch Metadata') {
-            withCredentials([file(variable: 'AWS_CONFIG_FILE',
```

Comment:
Hmm right, I forgot `AWS_CONFIG_FILE` is a well-defined env var that some of the stack understands. Should we have `withAWSBuildUploadCredentials` define that instead and update the bits which currently use AWS creds to drop the `--aws-config-file`-style arg when we can?

Reply:
I'd prefer to default to passing the file via an argument (rather than via an env var) everywhere since that's the lowest common denominator that works locally or in a remote session.